source/_integrations/zwave_js.markdown

```diff
@@ -1062,3 +1062,25 @@ No further action is required and the SmartStart product will be added automatic
 ### Terminology mapping table
 
 For some of the concepts, the terminology used in Home Assistant does not correspond to the terminology used in Z-Wave documentation. Refer to the [terminology mapping table](#z-wave-terminology-and-home-assistant) for a list of term equivalents.
+
+## Removing Z-Wave JS from Home Assistant
+
+This removes all paired Z-Wave devices and their entities, the Z-Wave JS add-on, and the Z-Wave integration from Home Assistant.
+
+### To remove Z-Wave JS from Home Assistant
+
+1. [Remove the device from your Z-Wave network](/integrations/zwave_js/#removing-a-device-from-the-z-wave-network).
+   - Do this for each device that is joined to your network so that it is no longer paired to the controller.
+   - You cannot add a device to a new controller while it is still paired with an old one.
+   - Alternatively, you can factory reset each device. Refer to the device manual to see how this is done.
+     - This usually involves finding the device in your household and pressing a button.
+2. Remove the Z-Wave JS add-on.
+   - Go to {% my supervisor_addon addon="core_zwave_js" title="**Settings** > **Add-ons** > **Z-Wave JS**" %}.
+   - Select **Uninstall**.
+   - Decide whether to also delete the data related to the add-on or whether to keep it.
+3. Remove the Z-Wave JS integration.
```

Comment:
We call the integration just `Z-Wave` nowadays. The add-on is still called Z-Wave JS.